### DISCUSSION TITLE
Integrate Corsair I-series powersupply values

### DISCUSSION THREAD

**Discussion by ShurikenGitHub** [timmo001/system-bridge#2457] [Ideas]
### Description

Hi, i have a Corsair HX1000i PSU in my pc. Via HWINFO i can read out the power/voltage/current/temp values from my powersupply. They come in via an internal USB-2 port connected to the PSU. Would it be possible to integrate these values into system bridge so i can read them out from home assistant? Would be awesome!

Thanks,
Stan

**Comment by timmo001** [comment]
All of libreharwaremonitor values should be outputted into the sensors module
